File: guildmasters/app/models/guild.rb
class Guild < ActiveRecord::Base
	belongs_to :guildmaster
	has_many :quests, dependent: :destroy
	has_many :adventurers, dependent: :destroy
	
	def self.get_info
	  guild = Guild.find(1)
	  return guild
	end
	
	def self.refresh
	  guild=Guild.find(1)
	  repeat = guild.popularity/20
	  repeat.times {
	    Adventurer.generate
	    Quest.generate
	    }
	end

end

File: guildmasters/app/assets/javascripts/models/guild.js
GM.Guild = DS.Model.extend({
	level: DS.attr('number'),
	popularity: DS.attr('number'),

	adventurers: DS.hasMany('adventurer'),
	quests: DS.hasMany('quest'),
	facilities: DS.hasMany('facility')
});

GM.GuildModel = DS.Model.extend();

GM.GuildModel.getAllGuilds = function () {
	$.ajax({
		type: 'GET',
	    url: 'guildsessions.json',
	    success: function(feedback) {
	    	var guild = feedback.guildsessions[0];
            GM.GuildmasterModel.guildmaster.guild = guild;
            console.log(feedback);
	    	GM.GuildmasterView = guildmasterTemplate(GM.GuildmasterModel.guildmaster);
	    	GM.GuildController.postGuildID(guild.id);
	    }
	});
}